### PR TITLE
Preparing for DCR 5% test

### PR DESCRIPTION
**Pull request by andre1050**
## What does this change?
Updates needed to test & variant names to run the "5% DCR" test.

## Why?
So we can separate the data from the current Frontend test.

### CODE REVIEW

File: src/web/components/SlotBodyEnd.tsx [hunk 1]
```diff
@@ -27,12 +27,12 @@ const sendOphanEvent = (action: OphanAction): void => {
...
-            campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
-            id: 'gdnwb_copts_memco_remote_epic_test_api',
+            campaignCode: 'gdnwb_copts_memco_dotcom_rendering_epic_api',
+            id: 'gdnwb_copts_memco_dotcom_rendering_epic_api',
         },
         abTest: {
-            name: 'remote_epic_test',
-            variant: 'remote',
+            name: 'dotcom_rendering_epic',
+            variant: 'api',
```
Comment (andre1050) [inline src/web/components/SlotBodyEnd.tsx]: The change from `remote` to `api` is intentional

File: src/web/components/SlotBodyEnd.tsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ const MemoisedInner = (props: Props) => {
 
     useEffect(() => {
         const contributionsPayload = buildPayload(props);
-        getBodyEnd(contributionsPayload)
+        getBodyEnd(contributionsPayload, 'http://localhost:8081/epic')
```
Comment (tjmw) [inline src/web/components/SlotBodyEnd.tsx]: Just adding a reminder that we don't want to accidentally merge this change :-)

Comment (andre1050) [reply src/web/components/SlotBodyEnd.tsx]: That's right 😬

File: src/web/layouts/CommentLayout.tsx [hunk 3]
```diff
@@ -187,10 +187,10 @@ export const CommentLayout = ({ CAPI, NAV }: Props) => {
 
     // Render the slot if one is true:
     // 1) The flag for this slot exists in the URL (i.e. ?slot-machine-flags=showBodyEnd)
-    // 2) The global switch for this slot is set to true;
+    // 2) The global switch for the Frontend/DCR Epic test is true
     const showBodyEndSlot =
         parse(CAPI.slotMachineFlags || '').showBodyEnd ||
-        CAPI.config.switches.slotBodyEnd;
```
Comment (andre1050) [inline src/web/layouts/CommentLayout.tsx]: This `slotBodyEnd` was initially added to give us an easy way to turn on/off the rendering of this slot. The name was meant to be future proof and therefore unaware of the content eventually loaded into that Slot (currently an Epic, in the future anything else).
For simplicity, I've decided to not look at this switch while we run the DCR 5% test and instead implement one and only one switch that's specific for this test. In the future we might bring back slot specific switches but doesn't feel like we need them yet.

**Pull request merged**